dashboard/src/actions/apps.ts

```diff
@@ -46,12 +60,74 @@ export type AppsAction = ActionType<typeof allActions[number]>;
 export function getApp(
   releaseName: string,
   namespace: string,
-): ThunkAction<Promise<void>, IStoreState, null, AppsAction> {
+): ThunkAction<Promise<hapi.release.Release | undefined>, IStoreState, null, AppsAction> {
```

Comment:
should this return an `IRelease`?

Reply:
Not in the case it throws an error.

Reply:
What I meant is, shouldn't `App.getRelease` return an `IRelease` so that this can return an `IRelease`?

Reply:
ah, I don't think so because that would mean that the returned object would contain the key `updateInfo` which will be always empty. To get an `IRelease` the method to be called would be `getAppWithUpdateInfo` (even though that method is now written to not return anything).